pkg/mdformatter/linktransformer/config.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Bartłomiej Płotka @bwplotka
+// Licensed under the Apache License 2.0.
+
+package linktransformer
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+)
+
+type Config struct {
+	Version int
+
+	Validate struct {
+		Validators []Validator `yaml:"validators"`
+	}
+}
+
+type Validator struct {
+	_regex  *regexp.Regexp
+	_maxnum int
+	// Regex for type github is reponame matcher, like `bwplotka\/mdox`.
```

Comment:
But what regex is for? Can we encode it in comment and field name?

Reply:
What about making a similar format as mdox web transformer config? So something like 

```
validators:
- glob: ..
  type: "github"    
```

Reply:
(first match wins)